packages/host/app/components/ai-assistant/panel.gts

```diff
@@ -445,12 +440,12 @@ export default class AiAssistantPanel extends Component<Signature> {
 
   @cached
   private get aiSessionRooms() {
-    let rooms: RoomField[] = [];
-    for (let resource of this.roomResources.values()) {
-      if (!resource.room) {
-        continue;
-      }
-      let { room } = resource;
```

Comment:
1 of the reasons we had resources as a map is to ensure all room members were settled. Becos, in the past sessions list we only return sessions which has invited and join members. 

Now if we move that away, I wonder if its ok to re-introduce a 'members' api

Reply:
Ok fine. 

After some deep digging into whether this can help the removal of room as a card. I have become more and more skeptical, mainly bcos, it requires me to add a lot of code to the matrix-handlers to index all the fields that room card had. 

I can appreicate the roomResource map pattern. Whats missing in the room resource:  
- is reacting to new events added
- fields within room card

Reply:
Hmmm ok I think we should put this PR in first. We can re-add resources map in panel maybe.